### ISSUE TITLE
how for MDI Frames, and MDI Child Dialogs !!!

### ISSUE THREAD

**Issue by esbvaGames** [wxFormBuilder/wxFormBuilder#782]
![wxFormBuilder_nICwbPl717](https://github.com/wxFormBuilder/wxFormBuilder/assets/5530499/9dfe2eb5-4a02-4961-a4c2-e3ffc440b253)
How can you create the MDI Frames, and the MDI Child dialogs, properly, because the parameter of the constructor here does not change, is there any example of how to use it making MDI Frames?

**Comment by sodevel** [comment]
wxFB does not support MDI, the subclass feature cannot be used because it cannot change the signature of the generated methods. Currently i don't see any other option but adding dedicated components for these widgets. However, im afraid the current type model isn't sufficient because of the special property of the child windows which must be added directly to a parent MDI window only.

TL;DR: This is quite some work for a feature Microsoft has declared to be obsolete so i have no plans to realize this. However, PR's are welcome.
